eldritch/events.py

```diff
@@ -1643,7 +1643,12 @@ def resolve(self, state):
       state.places[self.location_name].gate = None
 
     if self.seal_choice is None:
-      if self.character.clues < 5:  # TODO: this can also have modifiers
+      difficulty = 5
+      difficulty += state.ancient_one.get_modifier(self, 'difficulty')
```

Comment:
Use state.get_modifier so that it takes global effects into account too.

Reply:
Done.